rusk/tests/services/prover_service.rs

```diff
@@ -193,6 +193,13 @@ impl StateClient for TestStateClient {
     fn fetch_stake(&self, _pk: &PublicKey) -> Result<(u64, u64), Self::Error> {
         unimplemented!();
     }
+
+    fn fetch_existing_nullifiers(
+        &self,
+        _nullifiers: &[BlsScalar],
+    ) -> Result<Vec<BlsScalar>, Self::Error> {
+        Ok(vec![])
```

Comment:
Maybe add a comment/pending issue on why we are not checking the state in the tests?

Reply:
The prover service is not suppose to test those things, that is responsability of the state service test. However I agreed that it's a different mocking than the other methods, so I will make them consistent.